run.sh

```diff
@@ -1,24 +1,26 @@
-#!/bin/sh -l
+#!/usr/bin/env bash
+
+set -euo pipefail
 
 # inputs given by GitHub action -e docker runtime env vars
 TKTL_API_KEY=$INPUT_TKTL_API_KEY
 COMMIT_SHA=$GITHUB_SHA
 
 # Check
-tktl login $TKTL_API_KEY
+tktl login "$TKTL_API_KEY"
 
-until tktl get deployments -c $COMMIT_SHA ; do
+until tktl get deployments -c "$COMMIT_SHA" ; do
     sleep 10
     echo 'Waiting for public docs URL ...'
```

Comment:
This is probably cryptic for people.

Reply:
Good point. I think we can take this out entirely, will follow up separately.